core/src/toast/pane/pane.tsx

```diff
@@ -11,6 +11,7 @@ import s from "./pane.module.css";
 export interface ToastPaneType {
 	iconComponent: IconComponent;
 	iconCls: string;
+	style?: string;
```

Comment:
This should always be required, no optional.

Reply:
and btw it should be named "className". Style is common to be an object of raw CSS styles

Reply:
Anyway I think this should not be here. The "TYPE" in Pane is for negative/positive colouring, not for overall styling like shadow and border. Using ToastPane.styles.something directly would be better